apps/src/weblab2/CDOIDE/right-pane/JSPreview.tsx

```diff
@@ -0,0 +1,52 @@
+import {DebuggerWrapper} from '@cdoide/debugger/DebuggerWrapper';
```

Comment:
let's disable js execution for now

Reply:
Disabled by default via removing it from the `defaultPreviewFileTypesArray` value in `utils/previewFileTypes`

After this initial PR is in, I'll circle back and delete out the code that we're not using, I just wanted to get it checked into the repo and nicely compartmentalized into a PR we can undo later.